src/foremast/templates/configs/configs.json.j2

```diff
@@ -48,9 +48,9 @@
     },
     "qe": {
     },
-    "regions": [
```

Comment:
In terms of backwards compatibility, will there be any issues supporting the legacy format, or are we making a conscious upgrade?

Reply:
Disregard I see lines 105-109 in prepare_configs

Reply:
There should not be any issues. I am definitely trying to keep legacy support and in my testing, this still works.

The way we currently iterate over the list, works with iterating over keys in a dict so this didnt have any impact on old formats